ogre_mesh_viewer.py

```diff
@@ -139,6 +139,10 @@ def __init__(self, app):
         self.show_render_settings = False
         self.side_panel_visible = True
         self.is_filedialog_open = False
+        self.fixed_yaw_axes = {'X-up': (1, 0, 0), 'Y-up': (0, 1, 0), 'Z-up': (0, 0, 1)}
+        self.fixed_yaw_axis = 'Y-up'
```

Comment:
```suggestion
        self.fixed_yaw_axis = -1
```
where -1 means disabled, 0 means x-axis etc.

then construct the axis as
```py
axis = [0, 0, 0]
axis[self.fixed_yaw_axis] = 1
```

remove self.fixed_yaw